portal/templates/index.njk

```diff
@@ -33,6 +33,17 @@
 
     <div class="govuk-width-container">
 
+      {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
```

Comment:
I think I would expect all imports at the top of the file. Does that work here?

Reply:
I've moved it to the top level of `<script>` but I don't believe I can move it any higher than that